ops/opset13/cos.go

```diff
@@ -0,0 +1,76 @@
+package opset13
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/advancedclimatesystems/gonnx/onnx"
+	"github.com/advancedclimatesystems/gonnx/ops"
+	"gorgonia.org/tensor"
+)
+
+// Cos represents the ONNX cos operator.
+type Cos struct{}
+
+// newCos creates a new cos operator.
+func newCos() ops.Operator {
+	return &Cos{}
+}
+
+// Init initializes the cos operator.
+func (c *Cos) Init(attributes []*onnx.AttributeProto) error {
+	return nil
+}
+
+type CosDType interface {
```

Comment:
Maybe it is better to define `Float` typesets so we can use them for multiple operators? 

``` go 
type FloatType interface {
   float32 | float64
}
```